bin/workflows/pycbc_make_bank_verifier_workflow

```diff
@@ -179,24 +190,51 @@ inp_bank.description='TEMPLATEBANK'
 inp_bank.ifo_list = workflow.ifos
 inp_bank.segment = workflow.analysis_time
 
+# Inspinj Executable
+inspinj_exe = select_generic_executable(workflow, 'injection')
+# The output must be in xml format for pycbc_splitinj to work
+inspinj_exe.extension = ".xml"
+# The following line and the previous one are redundant for
+# lalapps_inspinj, but not for pycbc_create_injections
+inspinj_exe.current_retention_level = wf.Executable.FINAL_RESULT
+
 # Inspinj job
-inspinj_job = LalappsInspinjExecutable(workflow.cp, 'injection', out_dir='.',
-                                       ifos='HL', tags=[])
+inspinj_job = inspinj_exe(workflow.cp, 'injection', out_dir='.',
+                          ifos=workflow.ifos, tags=[])
 
 def add_banksim_set(workflow, file_tag, num_injs, curr_tags, split_banks):
     """Add a group of jobs that does a complete banksim.
     """
-    inspinj_job.update_current_tags(curr_tags)
     t_seg = segments.segment([1000000000, 1000000000+int(num_injs)])
-    node = inspinj_job.create_node(t_seg)
+    if 'LalappsInspinjExecutable' in str(inspinj_exe):
```

Comment:
I think you want `isinstance(inspinj_exe, LalappsInspinjExecutable)` for this

Reply:
I would say that this is the only remaining comment I would push back on. You are testing if the instance `inspinj_exe` is a certain class or not. `isinstance` is a builtin function for doing exactly this, casting to `str` and testing that feels messy.